src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -756,6 +800,9 @@ export default {
 				this.$kvTrackEvent('Monthly Digest', 'loan-feedback', 'dislike');
 			}
 		}
+
+		// Fetching MFI Recommendations Loans
+		this.fetchMFILoans();
```

Comment:
Same comment about whether this needs to be behind an experiment.

Reply:
Please Wrap this in the exp check